bcgov/mds
# MDS 4123 project description authorizations not clearing

# Main

- The previous authorizations on a project summary were not clearing when a new one was started

# How to test

- In MineSpace create a project summary for a mine, then create a second one and when you reach the authorizations step for it they should all be clear.
